includes/Services/SiteGenService.php

```diff
@@ -229,21 +231,15 @@ public static function complete( $active_homepage, $homepage_data ) {
 
 		// Update name and slug before generating child theme
 		$active_homepage = self::update_info_for_child_theme( $site_config, $active_homepage );
-		self::generate_child_theme( $active_homepage );
+		self::update_styles_for_sitegen( $active_homepage );
 
 		foreach ( $homepage_data as $index => $data ) {
```

Comment:
@amartya-dev Considering we aren't doing anything, can we skip this entire loop?

Reply:
yes :)

Reply:
I left it to set the activated home data in the flow data, this needs to be there for proper data persistence i think ? i am talking in reference to line 243, did not wanna introduce more bugs there